src/server/readpool/config.rs

```diff
@@ -49,113 +33,35 @@ pub struct Config {
 }
 
 impl Default for Config {
-    fn default() -> Config {
-        Config {
-            high_concurrency: DEFAULT_CONCURRENCY,
-            normal_concurrency: DEFAULT_CONCURRENCY,
-            low_concurrency: DEFAULT_CONCURRENCY,
-            max_tasks_high: DEFAULT_MAX_TASKS_PER_CORE * DEFAULT_CONCURRENCY,
-            max_tasks_normal: DEFAULT_MAX_TASKS_PER_CORE * DEFAULT_CONCURRENCY,
-            max_tasks_low: DEFAULT_MAX_TASKS_PER_CORE * DEFAULT_CONCURRENCY,
+    fn default() -> Self {
+        Self {
+            high_concurrency: 4,
```

Comment:
Why not keep `DEFAULT_CONCURRENCY`?

Reply:
because actually it is not used (in config files), removed to avoid confusion.

Reply:
I mean, none of the default values in config files relies on this value, and change it won't affect anything.

Reply:
Ok, I pushed a new commit, entirely removes "default()" (also there is no magic number 4 anymore).